nest/src/dashboard.service.ts

```diff
@@ -482,16 +482,16 @@ export class DashboardService {
             console.info("Found first bucket time: ", firstBucketTime)
             ds.forEach(d => datasets.push(d))
           }))
-        } else if (yAxis.type == "metrics") {
-          const va = yAxis as VerticalAxisMetrics;
-          promiseArray.push(await this.getVerticalAxisMetrics(runsPlus, input, va, sp).then(ds =>
```

Comment:
Oops, this promise aggregation was broken - should not have contained an await.
Still, is there a reason to drop the aggregation here entirely, rather than just dropping the await as you've done on line 492?

Reply:
Oh!
I see no reason to drop the aggregation.
In fact that should actually improve performance too
Thank you for pointing that out!